apps/site/util/imageUtils.ts

```diff
@@ -0,0 +1,13 @@
+export const isSvgImage = (src: string) => {
+  let isSvg = false;
```

Comment:
```suggestion
```

Instead of having a variable you can just `return false` at the end of function. And in your conditional state `return true`